pages/linux/edquota.md

```diff
@@ -0,0 +1,23 @@
+# edquota
+
+> Edit quota for a user or group.
+
+- Edit the user quota (default):
+
+`edquota --user {{user}}`
+
+- Edit the group quota:
+
+`edquota --group {{group}}`
+
+- Perform operations on given file system ONLY (default is to perform operations on all file systems with quota):
```

Comment:
Something tells me it'd be better to add this default info on a second line right after the description of the command itself.

```markdown
> Edit quota for a user or group.
> Operates on all file systems by default.
```

But @waldyrious is our UX expert here, let's see what he has to say :)